core/controllers/admin.py

```diff
@@ -848,3 +848,16 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class GetInteractionsByExplorationId(base.BaseHandler):
+    """This function returns a list of interaction id's for a exploration id."""
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        exploartion_id = self.request.get('exploration_id')
```

Comment:
Typo: ``exploration_id``

Reply:
@aggarwalpulkit596, is this comment resolved? [Please reply to all of the comments]

Reply:
yes done